examples/scenes/src/test_scenes.rs

```diff
@@ -1094,6 +1099,32 @@ mod impls {
         }
     }
 
+    pub(super) fn many_clips(scene: &mut Scene, params: &mut SceneParams) {
+        params.resolution = Some(Vec2::new(1000., 1000.));
+        let mut rng = StdRng::seed_from_u64(42);
+        let mut base_tri = BezPath::new();
+        base_tri.move_to((-50.0, 0.0));
+        base_tri.line_to((25.0, -43.3));
+        base_tri.line_to((25.0, 43.3));
+        for y in 0..10 {
+            for x in 0..10 {
+                let translate =
+                    Affine::translate((100. * (x as f64 + 0.5), 100. * (y as f64 + 0.5)));
+                const CLIPS_PER_FILL: usize = 3;
+                for _ in 0..CLIPS_PER_FILL {
+                    let rot = Affine::rotate(rng.gen_range(0.0..PI));
+                    scene.push_layer(Mix::Clip, 1.0, translate * rot, &base_tri);
+                }
+                let rot = Affine::rotate(rng.gen_range(0.0..PI));
+                let color = Color::rgb(rng.gen(), rng.gen(), rng.gen());
```

Comment:
This produces remarkably pleasing colours considering that it's completely random - it's probably a good technique to use in other examples.